scripts/recan_bundle_tdr.py

```diff
@@ -387,7 +387,7 @@ def main(argv):
         metadata = json.load(f)
 
     dss_source = DSSSourceRef(id='',
-                              name=SimpleSourceName(''))
+                              name=DSSSourceName(''))
```

Comment:
This can't work because `DSSSourceName` uses `kwonly=True`. PyCharm warns me about this. Please make sure you get a warning too and reply with a screenshot of that warning before addressing this. I want to make sure you use the right PyCharm inspection configuration.

Reply:
<img width="1303" alt="Screen Shot 2021-04-14 at 2 00 47 PM" src="https://user-images.githubusercontent.com/1947534/114778612-ee2bcd00-9d29-11eb-8a15-d1a6e81df5d8.png">